### ISSUE TITLE
Fix a11y contrast issues with code preview panels on documentation site

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#815]
Doing an axe-dev scan of component pages on the documentation site, some styles from the code preview panels is being flagged as non-compliant for contrast ratios.

```
Element has insufficient color contrast of 4.41 (foreground color: #ff5572, 
background color: #292d3e, font size: 9.9pt (13.2px), font weight: normal). Expected contrast ratio of 4.5:1
```
<img width="1267" alt="Screenshot 2023-03-16 at 9 15 47 AM" src="https://user-images.githubusercontent.com/45469809/225662580-a5dfb67b-ae1a-4369-9aa3-30d62f2b27d2.png">

Find the style that is causing the issue and update that color to be 4.5:1 or swap out with an accessible theme.

> **1**
